Initium-Odp/src/com/universeprojects/miniup/server/services/CombatService.java

```diff
@@ -37,4 +38,27 @@ public void enterCombat(CachedEntity attacker, CachedEntity defender, boolean au
 		
 		db.sendNotification(ds, defender.getKey(), NotificationType.fullpageRefresh);
 	}
-}
+	
+	/**
+	 * Removes all items on the current character for which the strength requirement
+	 * is no longer met (i.e. a buff is applied or expires).
+	 */
+	public void updateEquips(){
+		CachedEntity character = db.getCurrentCharacter();
```

Comment:
Actually I just realized in case of PvP, we will need to be able to specify a character (since the defender in a pvp scenario never actually issues his own server requests, he is merely defending automatically)